lib/archethic_web/aeweb/domain.ex

```diff
@@ -56,9 +58,9 @@ defmodule ArchethicWeb.AEWeb.Domain do
             ownerships: [ownership = %Ownership{secret: secret} | _]
           }} <- ReferenceTransaction.fetch_last(tx_address),
          {:ok, cert_pem} <- Map.fetch(json_content, "sslCertificate"),
-         %{extensions: extensions} <- EasySSL.parse_pem(cert_pem),
-         {:ok, san} <- Map.fetch(extensions, :subjectAltName),
-         ^domain <- String.split(san, ":") |> List.last(),
+         all_domain_names <-
+           get_all_domain_names(cert_pem),
```

Comment:
The function `get_all_domain_names` could be simplier using only EasySSL:
```suggestion
         %{all_domains: all_domain_names} <- EasySSL.parse_pem(cert_pem, all_domains: true),
```

Reply:
But we need to parse cert_pem to der format before giving it to EasySSL.parse_pem/2.

Reply:
EasySSL expose 2 functions: `parse_pem` and `parse_der`. Using the first one does not require to parse the cert to der format